src/components/organisms/settings/organization/createdOrganizationModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import { useTranslation } from "react-i18next";
+
+import { ModalName } from "@enums/components";
+import { useModalStore } from "@src/store";
+
+import { Button } from "@components/atoms";
+import { Modal } from "@components/molecules";
+
+export const CreatedOrganizationModal = () => {
```

Comment:
```suggestion
export const CreateNewOrganizationModal = () => {
```

Please change the name of the file as well.

Reply:
changed